tools/wake/main.cpp

```diff
@@ -527,33 +530,53 @@ int main(int argc, char **argv) {
     return 1;
   }
 
+  DescribePolicy policy = DescribePolicy::human();
+
+  if (tag) {
```

Comment:
Are these arranged in order of increasing detail? If not they probably should be.

Reply:
I generally agree with the sentiment but increasing detail isn't really defined here.
 
Is debug but not verbose more or less than verbose but not debug?
Is no-metadata with stdout/err more or less than metatdata without stdout/err